Library/Formula/boxes.rb

```diff
@@ -17,7 +17,7 @@ def install
 
     # distro uses /usr/share/boxes change to prefix
     system "make",
-      "GLOBALCONF=#{share}/boxes-config",
+      "GLOBALCONF=#{pkgshare}/boxes-config",
```

Comment:
Yes, I'll need to take a look at it and see if I can smash some of these issues a little harder.

The arm change should actually be:

```diff
diff --git a/Library/Formula/arm.rb b/Library/Formula/arm.rb
index e387899..a84c39c 100644
--- a/Library/Formula/arm.rb
+++ b/Library/Formula/arm.rb
@@ -7,8 +7,8 @@ class Arm < Formula
   bottle :unneeded
 
   def install
-    (share+"arm").install Dir["*"]
-    bin.write_exec_script share/"arm/arm"
+    pkgshare.install Dir["*"]
+    bin.write_exec_script pkgshare/"arm"
   end
 
   def caveats; <<-EOS.undent
@@ -16,7 +16,7 @@ class Arm < Formula
     See here for details: https://www.torproject.org/tor-manual.html.en
 
     To configure Arm, copy the sample configuration from
-    #{share}/arm/armrc.sample
+    #{pkgshare}/armrc.sample
     to ~/.arm/armrc, adjusting as needed.
     EOS
   end
```

But yes, it should not be flagging `pkgshare` followed by the `name` as incorrect. Aren't edge cases fun :wink: